### ISSUE TITLE
Source Code for Gen 3 Event Tool

### ISSUE THREAD

**Issue by WyrXIII** [suloku/wc-tool#4]
Like a few others I have the issue where my antirvirus will delete the programs because it deems them malicious; my workaround for this was to build the WC3 tool in Microsoft studio but I unfortunately can't find the source code for the Gen3 Event Tool 0.1d.exe . I'd really appreciate a response so I can turn back on my antivirus software while running it, cheers!

**Comment by suloku** [comment]
Yes, I've wanted to rebuiñd with visual studio for a while. Problem is there's an algorithm we are still unsure ee want to make publicly available. I'll rebuild with VS asap.

**Comment by WyrXIII** [comment]
Thanks man, will definitely keep my eye out for it.

**Comment by kamronbatman** [comment]
Barring this algorithm (which I don't think is a problem to release now), you can use the updated version.
